helm_deploy/values-dev.yaml

```diff
@@ -25,6 +25,7 @@ generic-service:
     COMPONENT_API_URL: 'https://frontend-components-dev.hmpps.service.justice.gov.uk'
     ACTIVITIES_API_URL: 'https://activities-api-dev.prison.service.justice.gov.uk'
     ENVIRONMENT_NAME: 'DEV'
+    SESSION_LIST_UI_DEFAULT_PAGINATION_PAGE_SIZE: "4"
```

Comment:
I've deliberately set the pagination size in dev to 4. This is to allow the team to test the pagination controls properly. 
There are only 15 overdue records in Wakefield, so with a pagesize of 50 they'd never be able to test pagination properly. 4 might seem like a weird number, but it is better than 3 or 5 which would result in exact page sizes. With 4 we should expect 3 pages of 4 records, plus a 4th page with 3 records on it.